Amazon.Ion.ObjectMapper/IonSerializer.cs

```diff
@@ -233,19 +249,19 @@ public void Serialize<T>(IIonWriter writer, T item)
                 return;
             }
 
-            if (item is System.Collections.IList) 
+            if (item is System.Collections.IList)

```

Comment:
I understand what's being done and why, but can we revert the changes for trailing whitespace and put them in a separate PR since it makes it very hard to see what's actually changed.

Reply:
Fixed